content/posts/linux/Desktop-Linux-Hardening.md

```diff
@@ -249,19 +249,44 @@ If you are using KickSecure or Whonix, most of these hardening have already been
 Note that these configurations do not disable unprivileged user namespaces. There are also a few things in `/etc/modprobe.d/30_security-misc.conf` to keep in mind:
 - The `bluetooth` and `btusb` kernel modules are disabled by default. You need to comment out `install bluetooth /bin/disabled-bluetooth-by-security-misc` and `install btusb /bin/disabled-bluetooth-by-security-misc` if you want to use Bluetooth.
 - Apple filesystems are disabled by default. This is generally fine on non-Apple systems; however, if you are using Linux on an Apple product, you **must** check what filesystem your EFI partition uses. For example, if your EFI filesystem is HFS+, you need to comment out `install hfsplus /bin/disabled-filesys-by-security-misc`, otherwise your computer will not be able to boot into Linux.
+- The `cdrom` and `sr_mod` modules are only blacklisted by default. If you have no intention to ever use CD-ROM devices they should be disabled. To implement this, at the bottom of the configuration file 'uncomment' both install (disable) commands and 'comment out' both existing blacklist commands.
```

Comment:
Technically right now no, but who knows what the kernel might do in the future if for some reason blacklist commands are pushed up the hierarchy. I will leave it you guys to decide.

Reply:
Fair point. My proposal to ignore the `blacklist` commands is in the interest of readability in an already very long post. If you can reword in a more digestible manner, then the part about commenting the `blacklist` commands can stand. Otherwise my recommendation is to leave out that part. (After all, accuracy at the time of writing is the only realistic expectation anyway.)

Reply:
I think let's not add this rn if its not the case